REQUIRE

```diff
@@ -3,3 +3,4 @@ DataStructures
 Distributions
 DataFrames
 Docile
+InfoZIP
```

Comment:
sorry I think is wrong, ignore this commit-- the next commit has the set up I believe you wanted